frontend/src/store/index.js

```diff
@@ -995,6 +995,9 @@ const getters = {
   canPatchShootsBinding (state) {
     return canI(state.subjectRules, 'patch', 'core.gardener.cloud', 'shoots/binding')
   },
+  canCreateKubeconfig (state) {
+    return canI(state.subjectRules, 'post', 'core.gardener.cloud', 'shoots/adminkubeconfig')
```

Comment:
```suggestion
    return canI(state.subjectRules, 'create', 'core.gardener.cloud', 'shoots/adminkubeconfig')
```

<img width="600" alt="image" src="https://github.com/gardener/dashboard/assets/1574023/a3a12ef7-42cf-441d-92d2-3572af558631">

Reply:
you are right, my bad for suggesting `post`